Concentus.Oggfile.sln

```diff
@@ -7,6 +7,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Concentus.Oggfile", "Concen
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OggTest", "OggTest\OggTest.csproj", "{812E353C-2404-4747-B7E6-1ADD162435BE}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "OpusTestForSeeking", "OpusTestForSeeking\OpusTestForSeeking.csproj", "{B526D4BB-65BC-4CD9-9B75-CC473EE4CB87}"
```

Comment:
Why add a separate testing project? What does this project need that the other test project doesn't have?